README.md

```diff
@@ -13,10 +13,81 @@ description: This is a sample Azure Function app created with the FastAPI framew
 ---
 <!-- YAML front-matter schema: https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main#supported-metadata-fields-for-readmemd -->
 
-# Using FastAPI Framework with Azure Functions
+# FastAPI on Azure Functions
+
+[![Open in GitHub Codespaces](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=brightgreen&logo=github)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=449261589)
+[![Open in Dev Containers](https://img.shields.io/static/v1?style=for-the-badge&label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/Azure-Samples/fastapi-on-azure-functions)
 
 Azure Functions supports WSGI and ASGI-compatible frameworks with HTTP-triggered Python functions. This can be helpful if you are familiar with a particular framework, or if you have existing code you would like to reuse to create the Function app. The following is an example of creating an Azure Function app using FastAPI.
 
+## Features
+- FastAPI integration with Azure Functions
+- Automatic OpenAPI/Swagger documentation
+- Python async support
+- Easy deployment to Azure
+- Built-in monitoring with Application Insights
+
+## Getting Started
+
+### Prerequisites
+- Python 3.9 or later
+- Azure Functions Core Tools
+- Azure CLI
```

Comment:
Azure CLI? Shouldnt it be azd? Or does it need both?

Reply:
it should be azd only